src/shared/tree/tree.scss

```
@@ -1,8 +1,10 @@
+@import '../global-styles/variables.scss';
+
 .tree-mount {
     position: fixed;
     top: 0;
     left: 0;
-    z-index: 2000;
+    z-index: $default-z-index;
```

From what I can tell, making the z-index be the same for both the hover overlay and the tree mount only fixes the problem because the hover overlay's mount is appended to the end of `document.body` but the tree mount is added further up the DOM tree? It's important to note that it doesn't actually need to be. It could just be added to the bottom making this fix not really deterministic. 

What about subtracting one for the default for this? Or just making them 2000 and 1999?
```suggestion
    z-index: $default-z-index - 1;
```

---

This is a good point in that restructuring the order of how this stuff gets injected could negate these changes. Pushing up the change.